mrf-paper/distributed_mrf.tex

```diff
@@ -74,9 +74,9 @@ \subsection{The Load Balancer Pattern}
 \begin{figure}[t]
   \centering
   % \includegraphics[width=0.8\linewidth]{fig/usl.png}
-  \begin{small}
-  \input{fig/usl.tex}
-\end{small}
+  % \begin{small}
+  % \input{fig/usl.tex}
```

Comment:
Seemed the fig/usl.tex is missing this is why it is commented out. Will change this once it is there